fauna/encoding/encoder.py

```diff
@@ -1,7 +1,7 @@
 from datetime import datetime, date
 from typing import Any, Optional, Set
 
-from fauna.query.models import DocumentReference, Module, Document, NamedDocument, NamedDocumentReference
+from fauna.query.models import DocumentReference, Module, Document, NamedDocument, NamedDocumentReference, Page
```

Comment:
`Page` is an unused import so that means it's missing from `_encode` ya?

Reply:
oh yeah, i added an explicit error for encoding, hence the import, but then removed it to let it error like any other unencodable obj

nice catch